Dockerfile

```diff
@@ -16,13 +16,19 @@ RUN yum -y update && \
 
 USER app
 WORKDIR /app
-
+RUN sudo yum -y install centos-release-scl-rh
+RUN sudo yum -y install devtoolset-3-gcc devtoolset-3-gcc-c++
```

Comment:
I think it would make sense for us to use the same build toolchain we use to create our lua_sandbox_extensions docker image. It's along the same lines as what you are doing here, but we are using devtoolset-6 there

https://github.com/mozilla-services/lua_sandbox_extensions/blob/dev/Dockerfile#L14.

https://github.com/mozilla-services/lua_sandbox_extensions/blob/dev/Dockerfile#L38